js/core/pci/scan.js

```diff
@@ -802,28 +803,8 @@ pciManager.each(function(pciDevice) {
   // });
 });
 
-// Print PCI devices debug info
-pciManager.each(function(pciDevice) {
-  var address = pciDevice.address();
-  var vector = pciDevice.getIRQVector();
-  var classData = pciDevice.classData();
-
-  var devicePin = 0;
-  if (!pciDevice.isBridge()) {
-    devicePin = pciDevice.interruptPin();
-  }
-
-  var pins = ['dont use', 'A', 'B', 'C', 'D'];
-
-  var info = address.bus.toString(16) + ':' + address.slot.toString(16) + '.' + address.func + ' ' +
-    pciDevice.vendorId().toString(16) + ':' + pciDevice.deviceId().toString(16) + ' ' +
-    classData.className + ' IRQ: ' + vector + ' PIN: ' + pins[devicePin];
-  debug(info);
-});
-
```

Comment:
Just use the js folder from this PR as the runtimejs dependency for the initrd and use the debug option as true in the package.json. When you start it, it should hang after the pci debug info.